DQM/Physics/interface/Hemisphere.hh

```diff
@@ -0,0 +1,202 @@
+#ifndef Hemisphere_h
```

Comment:
could this just use https://github.com/cms-sw/cmssw/blob/0397259dd747cee94b68928f17976224c037057a/PhysicsTools/PatAlgos/interface/HemisphereAlgo.h
which looks to be a modernized version of this code